src/registrar/models/domain_application.py

```diff
@@ -501,6 +501,18 @@ def _send_status_update_email(
         except EmailSendingError:
             logger.warning("Failed to send confirmation email", exc_info=True)
 
+    def set_approved_domain(self):
+        """Always called in parallel to the approve method"""
```

Comment:
```suggestion
        """Always called in parallel to the approve method and is used to link the domain as a foreign key in the application"""
```
added a but more detail on the purpose here